src/js/components/Paragraph/Paragraph.js

```diff
@@ -17,9 +17,9 @@ class Paragraph extends Component {
 }
 
 if (process.env.NODE_ENV !== 'production') {
-  doc(Paragraph);
+  documentParagraph(Paragraph);
 }
 
-export default compose(
+export const ParagraphWrapper = compose(
```

Comment:
I wonder that is the display name for this. maybe we can do `class ParagraphComponent` and `const Paragraph` so that the display name is Paragraph.

Reply:
I think we want the `displayName` to be the non-hoc component. Otherwise, most of the console messages will show "ParagraphComponent". I was thinking it was better to have the HOC be the odd name, but to allow the import to look right by mapping the HOC name in index.js.